### DISCUSSION TITLE
What criteria is used to determine the weather icons?

### DISCUSSION THREAD

**Discussion by cnlane** [weather-gov/api#204] [Q&A]
Using this url as an example, an icon is recommended.  (They are all really lovely, by the way)

https://api.weather.gov/gridpoints/BOU/62,61/forecast

recommends:

"icon": "https://api.weather.gov/icons/land/day/tsra,60?size=medium",

Do you have a document describing the criteria for each weather icon such as 'if there was rain and the temperature was between 20 and 35 F, return rain_frza.png'?  The icons are very self-explanatory but I'm writing code to emulate what you did and would appreciate your insights.

**Comment by scadergit** [comment]
There are NWS directives that describe how the short and long description should generate from the forecast data. I would provide the link, but the server they're on is not behaving after the building they are hosted in flooded. Perhaps @StephenClouse has another reference to the documents. The generated phrases are also influenced by configuration options set by each office.
